### PR TITLE
Closes #10782: Update crash reporting doc with what information breadcrumbs are recording

### LINKED ISSUE #10782: FNX-4555 ⁃ Update crash reporting doc with breadcrumb information
Add a place to track what breadcrumbs are we sending during a crash report.

### PR DESCRIPTION
**Pull request by rocketsroger**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: docs/crash-reporting.md [hunk 1]
```diff
@@ -12,6 +12,14 @@ Documentation for the specific libraries is included in the [Android Components
 
 The Glean crash ping format is documented [here](https://github.com/mozilla-mobile/android-components/blob/master/components/lib/crash/docs/metrics.md).
 
+## Breadcrumbs
+
+[Breadcrumbs](https://github.com/mozilla-mobile/android-components/blob/master/components/support/base/src/main/java/mozilla/components/support/base/crash/Breadcrumb.kt) are trail of events that are sent with each crash report to both Socorro and Sentry.
```
Comment (liuche) [inline docs/crash-reporting.md:17]: To make this documentation clearer, can you also include that the breadcrumb includes:

File: docs/crash-reporting.md [hunk 2]
```diff
@@ -12,6 +12,14 @@ Documentation for the specific libraries is included in the [Android Components
 
 The Glean crash ping format is documented [here](https://github.com/mozilla-mobile/android-components/blob/master/components/lib/crash/docs/metrics.md).
 
+## Breadcrumbs
+
+[Breadcrumbs](https://github.com/mozilla-mobile/android-components/blob/master/components/support/base/src/main/java/mozilla/components/support/base/crash/Breadcrumb.kt) are trail of events that are sent with each crash report to both Socorro and Sentry.  
+
+### Events
+
+In [HomeActivity](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/HomeActivity.kt) when `onDestinationChanged` occurs, the destination fragment's name and its custom tab value is added to the breadcrumbs.
```
Comment (liuche) [inline docs/crash-reporting.md:21]: "and whether it is a custom tab" (to make it clear it's a boolean)

**Comment by liuche** [comment]
(in the future, you don't need to file the data review in bugzilla, but can just include it in the PR, like #10785 - that will keep all this documentation/code changes together so it's easy to find in the future)

File: docs/crash-reporting.md (hunk 2 above)
Comment (rocketsroger) [reply docs/crash-reporting.md:21]: Thanks, I'll fix.

**Pull request merged**